app/serializers/analytics_dashboard_organizer_serializer.rb

```diff
@@ -0,0 +1,104 @@
+class AnalyticsDashboardOrganizerSerializer < ActiveModel::Serializer
+  # has_many :dreams
+  attributes :good_dreams, :dream_dates, :pie_chart, :stacked_bar
+
+  def good_dreams
+    object.dreams.map do |dream|
+      DreamSerializer.new(dream)
+    end
+  end
+
+  def dream_dates
+    dates = []
+    good_dreams.each do |dream|
+      dates << dream.nice_date
+    end
+    dates
+  end
+
+  def pie_chart
+    @_pie_chart_data ||= begin
+    [{
+          name: "Positive",
+          y: positive_result
+      }, {
+          name: "Negative",
+          y: negative_result,
+          sliced: true,
+          selected: true
+      }, {
+          name: "Neutral",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.